topic-configuration/README.adoc

```diff
@@ -54,6 +54,8 @@ ifdef::context[:parent-context: {context}]
 [role="_abstract"]
 As a developer of applications and services, you can refer to the properties of your topics in {product} to better understand how the Kafka implementation is managed for your services. You can edit certain topic properties according to the needs and goals of your services. Kafka topics contain the data (events) that applications produce or consume, so the way the topics are configured affects how data is stored and exchanged between applications.
 
+In addition, as a developer, you can use the {product} web console to check Kafka topics in {product} for matching schemas in {registry-product-long}. When you use a schema with your Kafka topic, the schema ensures that producers publish data that conforms to a certain structure and compatibility policy. The schema also helps consumers parse and interpret the data from a topic as it is meant to be read. Using the web console to quickly identify matches between Kafka topics and schemas means that you or others in your organization don't need to inspect client application code to check for schema usage. If you don’t have any Service Registry instances, or don’t have a schema that matches your topic, the console provides links to Service Registry, where you can create these.
```

Comment:
Suggest the following for consistency of language to help avoid confusion: 
- Change "matching schemas..." to "associated schemas..."
- Change "quickly identify matches..." to "quickly identify associations..."  
- Change "schema that matches your topic... to "schema associated with your topic..."

Also suggest using {registry} attribute instead of Service Registry (twice in last sentence)

Reply:
Per https://github.com/redhat-developer/app-services-guides/pull/344#discussion_r740392068, will lean towards _match_, where possible.